tethys_cli/gen_templates/portal_config

```diff
@@ -1,9 +1,37 @@
 # Portal Level Config File
 
-# see tethys documentation for how to setup this file
-version: 1.0
-name:
-apps: {}
-settings:
-  SECRET_KEY: {{ SECRET_KEY }}
-site_content: {}
\ No newline at end of file
+# For more information about this file refer to the Tethys Docs (http://docs.tethysplatform.org/en/stable/tethys_portal/configuration.html)
+version: {{ version }}
+name: {{ name }}
+
+############################################################################
+#
+# APPS
+#
+# This section is applied when running the "tethys install" command. Tethys
+# will first look for app settings in this file. If none are found it will
+# then look for a "services.yml" file in the install location.
+#
+#############################################################################
```

Comment:
I'm not a huge fan of these bulky comment blocks, though I think the comments could be helpful. What do you think about making them a little less bulky:

```
############################################################################
# APPS
# The "tethys install" command checks this section for app settings. 
############################################################################
{{ apps }}
```

Reply:
I like having them be less bulky. My main concern is that it was confusing that each section of the `portal_config.yml` was applied at different times and in different ways. I was hoping having the comments in the file itself would help clarify how each section is used. I'll adjust them to be more concise.